.env.github-api

```diff
@@ -0,0 +1,8 @@
+GITHUB_CLIENT_ID=aac2c89a1b7f5749c3db
```

Comment:
@joshwyatt I'm not sure why this change shows up here., and a similar change shows up in @SSabu's PR. These have already been merged as part of the mob code session as far as I know. Do you know what happened or what to do with it?